### ISSUE TITLE
Why is Anaconda needed on Linux?

### ISSUE THREAD

**Issue by scotus-49** [wassimj/topologicsverchok#6]
From the setup file:
```
if platform == 'linux' or platform == 'linux2' or platform == 'darwin':
  conda = '/opt/anaconda3/envs'
```
My setup:
System: Ubuntu 21.04
Blender: 2.93.0 Beta (Appimage)
Blender Python version: 3.9

Why is it necessary to install Anaconda to run this addon? I have already installed topologicPy with no problems, but your __init__.py setup won't run without Anaconda. The setup is also looking for the /opt directory in my home directory:
```
FileNotFoundError: [Errno 2] No such file or directory: '/home/scotus/opt/anaconda3/envs'
```

**Comment by wassimj** [comment]
Hi @scotus-49. It is just how I set it up because sometimes the Blender python version is different from the system python version. If you know what you are doing, you can modify the init.py file to suit your environment. Do you know how to do this or do you need help?

**Comment by scotus-49** [comment]
Thanks for your quick reply. I am not an expert on Blender Python and the addon system, so any help would be much appreciated.

I assume that the addon needs to find the topologicPy with the egg file, and I"m looking into the init.py code, and trying to find where it's looking for it. I installed  topologicPy into ~/topologicbim on my system, as recommended.

**Comment by scotus-49** [comment]
The add-on now works for me with Anaconda, on my Ubuntu 21.04 with the 2.93.0 Beta Appimage from Blender. 

I made a minor alteration to the python install script to account for the location of the 'site-packages' folder in my Blenderxxx Anaconda envs folder, and manually copied the topologic-0.4-py3.9.egg file from ~/topologicbim/TopologicPy/cpython/dist. I renamed the Blender python folder in the Appimage installation setup, but did not copy over the Anaconda env Blenderxxx folder as you suggested. Activating the conda Blender environment in the terminal before starting Blender seemed to work fine.

There's still a whole list of errors coming up in the terminal, but Blender let me activate the addon, and it shows up as a submenu under Sverchok. I still have to find and install some missing libraries for Sverchok, but Anaconda makes the job a bit easier.

![topologic-sverchok blender addon](https://user-images.githubusercontent.com/64618119/116013624-7adf6080-a631-11eb-8585-54de90b80bc8.png)

Now the question is - what can I do with it?
